lib/PhasarLLVM/Utils/phasar_llvm_utils-config.cmake

```diff
@@ -3,7 +3,10 @@ set(PHASAR_phasarllvm_utils_COMPONENT_FOUND 1)
 list(APPEND
   PHASAR_LLVM_DEPS
   Support
-  Core)
+  Core
+  BitWriter
+  Demangle
+)
```

Comment:
for some reason here we still have this file, in other places we don't. forgot this one?